TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -729,8 +729,11 @@ def makeTreeFromMiniAOD(self,process):
     process.load("RecoBTag.SecondaryVertex.candidateBoostedDoubleSecondaryVertexAK8Computer_cfi")
     process.load("RecoBTag.SecondaryVertex.pfBoostedDoubleSecondaryVertexAK8BJetTags_cfi")
 
+    from TreeMaker.Utils.deepak8producer_cfi import DeepAK8Producer
+    process.deepAK8 = DeepAK8Producer.clone()
+
     # add discriminator and update tag
-    process, JetAK8Tag = addJetInfo(process, JetAK8Tag, [], [], cms.VInputTag(cms.InputTag("pfBoostedDoubleSecondaryVertexAK8BJetTags")))
+    process, JetAK8Tag = addJetInfo(process, JetAK8Tag, [], [], cms.VInputTag(cms.InputTag("pfBoostedDoubleSecondaryVertexAK8BJetTags")), ['deepAK8'])
```

Comment:
should be:
```
process, JetAK8Tag = addJetInfo(process, JetAK8Tag, ['deepAK8:tDiscriminatorDeep','deepAK8:wDiscriminatorDeep'], [], cms.VInputTag(cms.InputTag("pfBoostedDoubleSecondaryVertexAK8BJetTags")))
```
but this will only work once the producer outputs userfloats instead of `vector<double>`